wagtail/images/tests/tests.py

```diff
@@ -183,10 +182,13 @@ def setUp(self):
             'test name',
             'test label',
             'test classnames',
-            'test filter spec'
+            'original'
         )
         # test image
-        self.image = MagicMock()
+        self.image = Image.objects.create(
+            title="Test image",
+            file=get_test_image_file(),
+        )
         self.image.id = 0
```

Comment:
Looks like these tests are currently failing on database backends other than sqlite. It may well be that `self.image.id = 0` can simply be removed now...